### DISCUSSION TITLE
Vue Storefront

### DISCUSSION THREAD

**Discussion by martijnvdbrug** [vendurehq/vendure#694] [General]
Hello everyone! I have seen a lot of questions and answers about Vue Storefront in Slack. Maybe it is a good idea to bundle all information related to Vue Storefront here, so it will stay available?

Do you know any live websites that use Vendure + Vue Storefront?
Do you know of any sample code or public repositories using Vendure + Vue Storefront?
Is there a Vendure integration available, or is it easier to implement it yourself?
What about Vendure plugins vs. Vue Storefront? (Custom graphql fields etc)

**Comment by michaelbromley** [comment]
The only example I know of right now with VSF is https://cbmadeinitaly.com/. The author made some custom adapter for VSF 1.x, which unfortunately is not open source.

I've not attempted an integration myself, but I did ask the VSF team about how to go about it and they directed me to https://docs.vuestorefront.io/v2/integrate/integration-guide.html

It would be cool if someone wants to build a VSF Next/Vendure integration. If so, I'm happy to advise!

**Reply by martijnvdbrug** [reply]
I'm interested in helping with development if anyone already started on a VSF integration. If not, I will attempt it for any upcoming projects.

**Comment by martijnvdbrug** [comment]
Since I already have a small framework setup with Gridsome (Vue static site generator), I am also playing around a bit with just the UI layer of Vue Storefront: www.storefrontui.io  which is just the Vue components. Free for everyone!

**Comment by michaelbromley** [comment]
Update: I've had a meeting with the co-founder & CTO of Vue Storefront and it is likely that we will be having a partnership to make VSF an officially-supported integration for Vendure. More news hopefully in the coming weeks.

**Reply by michaelbromley** [reply]
Work has now started on an integration: https://github.com/vuestorefront/vendure

There is a dedicated channel on the [Vue Storefront Discord](http://discord.vuestorefront.io/) where progress and implementation can be discussed.
